sfepy/tests/test_declarative_examples.py

```diff
@@ -25,6 +25,7 @@ def inedir(filename):
     'diffusion/sinbc.py',
     'diffusion/time_advection_diffusion.py',
     'diffusion/time_poisson.py',
+    'diffusion/time_heat_equation_multi_material.py',
```

Comment:
Put above time_poisson.py to preserve alphabetical order?

Reply:
Alright!